services/src/main/scala/cromwell/services/healthmonitor/impl/CommonMonitoredSubsystems.scala

```diff
@@ -0,0 +1,46 @@
+package cromwell.services.healthmonitor.impl
+
+import cromwell.core.docker.{DockerCliClient, DockerCliKey}
+import cromwell.services.EngineServicesStore
+import cromwell.services.healthmonitor.HealthMonitorServiceActor.{MonitoredSubsystem, OkStatus, SubsystemStatus}
+
+import cats.instances.future._
+import cats.syntax.functor._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * A mixin which provides checks which should be standard across implementations.
+  */
+trait CommonMonitoredSubsystems {
```

Comment:
I couldn't remember why I changed this from an `Object` to a `Trait` and it seems more natural to be an `Object`. I played around with it a bit and it was due to the need for the `ExecutionContext`. It started being a game of "which suboptimal path do I want to follow" and then I decided that with only 2 impls right now it doesn't really matter

Reply:
done